### PR TITLE
#201 Report lack of disk space more nicely

### LINKED ISSUE #201: Report lack of disk space more nicely
```
local_publish > cannot upload package 'aem/home/lib/foo.ui.content.live-6.0.0-SNAPSHOT.zip'; unexpected status: Inaccessible value 
```

### PR DESCRIPTION
**Pull request by piotr-andruszkiewicz-wttech**
fixes #201

### CODE REVIEW

File: pkg/package_manager.go [hunk 1]
```diff
@@ -423,12 +423,19 @@ func (pm *PackageManager) uploadBuffered(localPath string) (string, error) {
...
 	if !status.Success {
-		return "", fmt.Errorf("%s > cannot upload package '%s'; unexpected status: %s", pm.instance.ID(), localPath, status.Message)
+		return "", fmt.Errorf("%s > cannot upload package '%s'; %s", pm.instance.ID(), localPath, pm.interpretFail(status.Message))
 	}
 	log.Infof("%s > uploaded package '%s'", pm.instance.ID(), localPath)
 	return status.Path, nil
 }
 
+func (pm *PackageManager) interpretFail(message string) string {
+	if strings.EqualFold(message, "Inaccessible value") {
```
Comment (krystian-panek-wttech) [inline pkg/package_manager.go:433]: if I were you I would check this using contains() and tolowercase() ;)

File: pkg/package_manager_test.go [hunk 2]
```diff
@@ -0,0 +1,51 @@
+package pkg
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestPackageManager_interpretFail(t *testing.T) {
```
Comment (krystian-panek-wttech) [inline pkg/package_manager_test.go:8]: I don't see a value of this test.. what's your motivation?

**Review by krystian-panek-wttech — CHANGES_REQUESTED** [review]
see comms

File: pkg/package_manager_test.go (hunk 2 above)
Comment (piotr-andruszkiewicz-wttech) [reply pkg/package_manager_test.go:8]: It's the best I can do without killing my local setup for testing 😄

Comment (krystian-panek-wttech) [reply pkg/package_manager_test.go:8]: this test is testing string operation; it's too obvious, sry

**Review by krystian-panek-wttech — APPROVED** [review]

**Pull request merged**